content/ri/release-notes/v2.10.0.md

```diff
@@ -0,0 +1,37 @@
+- [#1159](https://github.com/RedisInsight/RedisInsight/pull/1159), [#1160](https://github.com/RedisInsight/RedisInsight/pull/1160), [#1068](https://github.com/RedisInsight/RedisInsight/pull/1068), [#1071](https://github.com/RedisInsight/RedisInsight/pull/1071), [#1095](https://github.com/RedisInsight/RedisInsight/pull/1095), [#1097](https://github.com/RedisInsight/RedisInsight/pull/1097), [#1098](https://github.com/RedisInsight/RedisInsight/pull/1098) Open a dedicated "Analysis Tools" page which displays the enhanced overview for Cluster databases to see memory and key allocation, as well as monitor database information per shards
+- [#1017](https://github.com/RedisInsight/RedisInsight/pull/1017), [#1025](https://github.com/RedisInsight/RedisInsight/pull/1025), [#1029](https://github.com/RedisInsight/RedisInsight/pull/1029), [#1059](https://github.com/RedisInsight/RedisInsight/pull/1059), [#1092](https://github.com/RedisInsight/RedisInsight/pull/1092) Add support for additional data formats in Browser/Tree view, including Protobuf, Binary, Pickle, PHP unserialize (view and edit serialized PHP values as JSON), Java serialized objects
+- [#1130](https://github.com/RedisInsight/RedisInsight/pull/1130) Save formatters selected for other keys
+- [#1177](https://github.com/RedisInsight/RedisInsight/pull/1177) Add a validation when edited value is not valid in the selected format in Browser/Tree view
+- [#1048](https://github.com/RedisInsight/RedisInsight/pull/1048) Ability to configure Workbench to not clear the Editor after commands have been run
+- [#1119](https://github.com/RedisInsight/RedisInsight/pull/1119) Move the Pipeline mode configuration for Workbench to the dedicated "Workbench" section in settings
+- [#1149](https://github.com/RedisInsight/RedisInsight/pull/1149) Group results in Workbench to see many results in one box to save space
+- [#1162](https://github.com/RedisInsight/RedisInsight/pull/1162) Add a link with User Survey
+- [#1037](https://github.com/RedisInsight/RedisInsight/pull/1037) Add tooltips for [Redis Streams](https://redis.io/docs/data-types/streams/) to display long fields
+- [#1202](https://github.com/RedisInsight/RedisInsight/pull/1202) Removed format validations from the admin username in the Redis Enterprise Cluster autodiscovery process
+- [#1179](https://github.com/RedisInsight/RedisInsight/pull/1179) Updated a message when there are no keys in Browser/Tree view to provide guidances on how to quickly add test data to Redis database
+
+**Bugs**
+- [#1180](https://github.com/RedisInsight/RedisInsight/pull/1180) Truncate TTL in minutes
+- [#1197](https://github.com/RedisInsight/RedisInsight/pull/1197) Fix a case when Workbench is not available due to the failed encryption
+- [#1176](https://github.com/RedisInsight/RedisInsight/pull/1176) Save the refresh value in Browser/Tree view
+- [#1101](https://github.com/RedisInsight/RedisInsight/pull/1101) Fix an issue when key names are not displayed 
```

Comment:
Enable key name display?

Reply:
it was a very rare case when key names were not displayed, let's leave it as is?

Reply:
sounds good to me